pkg/mirrorregistries/mirror_registry_test.go

```diff
@@ -129,6 +130,19 @@ var _ = Describe("Generator tests", func() {
 				res := m.IsMirrorRegistriesConfigured()
 				Expect(res).Should(Equal(true))
 			})
+
+			It("returns true when only the system CA exists (and registry.conf has mirrors)", func() {
```

Comment:
I don't understand this test. Why is there a conditional and why does `IsMirrorRegistriesConfigured` have anything to do with the system CA, shouldn't it be based on the user one?

Reply:
Yeah, fair enough. So added a new set of tests for GetMirrorCA.